applications/nrf_desktop/src/modules/dfu_mcumgr.c

```diff
@@ -75,6 +75,29 @@ static void dfu_timeout_handler(struct k_work *work)
 	}
 }
 
+static bool smp_cmd_is_dfu_cmd(const struct mgmt_evt_op_cmd_arg *cmd)
+{
+	const char *smp_cmd_name = "Unknown";
+	bool dfu_transfer_cmd = true;
+
+	if (cmd->group == MGMT_GROUP_ID_IMAGE) {
+		smp_cmd_name = "Image Management";
+	} else if ((cmd->group == MGMT_GROUP_ID_OS) && (cmd->id == OS_MGMT_ID_RESET)) {
+		smp_cmd_name = "OS Management Reset";
+	} else if (IS_ENABLED(CONFIG_MGMT_SUITFU_GRP_SUIT) &&
+		   (cmd->group == CONFIG_MGMT_GROUP_ID_SUIT)) {
+		smp_cmd_name = "SUIT Management";
+	} else {
+		dfu_transfer_cmd = false;
+	}
+
+	if (dfu_transfer_cmd && smp_cmd_name) {
```

Comment:
Wouldn't we want to know when debugging what exactly group was send? Also wouldn't we want to know that some other group had been send? 

I suggest dropping if statement and LOG_DBG each time with event name and group ID.

Reply:
We already log incoming events at the beginning of MCUmgr handler

Reply:
Yes :) I meant to log it all here in one place and then do not check if (dfu_transfer_cmd && smp_cmd_name)